hyperspy/tests/io/protochips_data/invalid_protochips_file.csv

```diff
@@ -12,4 +12,4 @@ Time, Notes, Channel A Current, Channel A Voltage, Channel A Resistance, Channel
 196, , 0.0179984, 39.9278, 2218.40830296026, 0, 0, NaN
 296, , 0.0179992, 39.7102, 2206.22027645673, 0, 0, NaN
 396, , Test, 39.5232, 2195.79432762021, 0, 0, NaN
-
+497, , 0.0179994, 39.3669, 2187.1229040968, 0, 0, NaN
```

Comment:
Why was the absence of this line raising a warning?

Reply:
For this line: https://github.com/hyperspy/hyperspy/blob/24619445e501cbc541d1b4179db31337d4959a8f/hyperspy/io_plugins/protochips.py#L176

Actually, there are two bugs here... for some files, the first line of data is skipped... and the slicing should be [1:-1] to avoid taking the average with the edge values.

Reply:
It is worth then opening a separate issue to report the bug.

Reply:
Fixed.